packages/xo-server/src/xapi/index.js

```diff
@@ -1406,11 +1406,14 @@ export default class Xapi extends XapiBase {
           table.grainLogicalAddressList,
           table.grainFileOffsetList
         )
-        await this._importVdiContent(vdi, vhdStream, VDI_FORMAT_VHD)
-
-        // See: https://github.com/mafintosh/tar-stream#extracting
-        // No import parallelization.
-        cb()
+        try {
+          await this._importVdiContent(vdi, vhdStream, VDI_FORMAT_VHD)
+          // See: https://github.com/mafintosh/tar-stream#extracting
+          // No import parallelization.
+          cb()
+        } catch (e) {
+          reject(e)
```

Comment:
Shouldn't we still cause `cb` in case of rejection?

Reply:
why not, which one should be called first? `ce(e)` or `reject(e)`

Reply:
The order does not matter :slightly_smiling_face: